src/sched_rebuild.c

```diff
@@ -38,9 +38,35 @@ struct sched_class *mod_class[] = {
 	&shadow_idle_sched_class,
 };
 
+DEFINE_PER_CPU(struct list_head, dying_task_list);
+
 #define NR_SCHED_CLASS 5
 struct sched_class bak_class[NR_SCHED_CLASS];
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 3, 0)
+static void put_prev_task_fake(struct rq *rq, struct task_struct *prev)
+{
+}
+
+static const struct sched_class fake_sched_class = {
+	.put_prev_task = put_prev_task_fake,
+};
+
+static struct task_struct fake_task = {
+	/*
+	 * Avoid pull_{rt,dl}_task()
+	 */
+	.prio = MAX_PRIO + 1,
+	.sched_class = &fake_sched_class,
+};
+
+#define pick_next_task_rq(class) \
+	(class)->pick_next_task(rq, &fake_task, &rf)
+
+#elif
+#define pick_next_task_rq(class) \
+	(class)->pick_next_task(rq)
+#endif
```

Comment:
I think this looks better. Because it's more clear that `rf` is not an unused variable.
```c
#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 3, 0)
#define pick_next_task_rq(class, rf) \
	(class)->pick_next_task(rq, &fake_task, &(rf))
#else
#define pick_next_task_rq(class, rf) \
	(class)->pick_next_task(rq)
#endif
```

Reply:
fake_task too